apps/api/src/app/invites/invites.controller.ts

```diff
@@ -72,4 +72,19 @@ export class InvitesController {
 
         return mapEntity(invite)
     }
+
+    @Post("redeem/:code/group/:group")
+    @ApiOperation({ description: "Redeems a specific invite." })
+    @ApiCreatedResponse({ type: InviteResponse })
+    async redeemInvite(
```

Comment:
I think we need to add security to this new endpoint since people can call it from the dashboard, API docs or using the API SDK. Then we would need to add the same logic as the other endpoints that modify the database information.

Such as [create invite](https://github.com/bandada-infra/bandada/blob/main/apps/api/src/app/invites/invites.controller.ts#L39) and [update group](https://github.com/bandada-infra/bandada/blob/main/apps/api/src/app/groups/groups.controller.ts#L181). All the functions that change the database. All those functions have a `<function-name>WithApiKey` and `<function-name>Manually`. 

What do you think?

Reply:
Yeah makes sense, will update it now.